apis/infrastructure/v1beta1/webhook_suite_test.go

```diff
@@ -96,13 +98,21 @@ var _ = BeforeSuite(func() {
 		MetricsBindAddress: "0",
 	})
 	Expect(err).NotTo(HaveOccurred())
+	user, err := testEnv.ControlPlane.AddUser(envtest.User{
+		Name:   "test-user",
+		Groups: []string{"system:byoh"},
+	}, nil)
+	Expect(err).NotTo(HaveOccurred())
 
-	err = (&byohv1beta1.ByoHost{}).SetupWebhookWithManager(mgr)
+	testUserK8sClient, err = client.New(user.Config(), client.Options{Scheme: scheme.Scheme})
```

Comment:
Question: Is this code added for future use? The default group will be `"byoh":"hosts"` in the CSR generation in the other PR.

Reply:
updated.